splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/DB2VarcharCompatibilityIT.java

```diff
@@ -480,4 +495,30 @@ public void testParameterizedMultiProbeScan() throws Exception {
         testPreparedQuery1(sqlTemplate, expected, false, true);
 
     }
+
+    @Test
+    public void testSparkDistinct() throws Exception {
+        String query = format("select distinct v1, v2 from z --splice-properties useSpark=true", useSpark);
```

Comment:
Can you please change useSpark=true to useSpark=%s so we test control also?

Reply:
Sure, it's a copy-pasting error and fixed in the latest commit.